INN/umbrella-currentorg
# LTW projects ordering

## Changes

This pull request makes the following changes:

- Adds an `orderby` arg to the arguments in the `projects-base.php` partial to order the projects by descending dates and ascending titles, so that they're order like this:

```
A 2020
B 2020
C 2020
A 2019
B 2019
C 2019
```
- Adds in the project year above the title in the list item partial so we don't have to click each project to see its year and verify it's in the correct order - **this needs to be deleted before merging**

## Why



For #160

## Testing/Questions

Features that this PR affects:

- LTW projects and how they're sorted/ordered on the frontend


Questions that need to be answered before merging:

- [ ] Does it work as expected?
- [ ] Any weird edge cases?

Steps to test this PR:

1. View the LTW projects page and make sure all posts are sorted like the example above, aka by year alphabetically
